src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -43,14 +43,13 @@ object Boot extends App {
         swaggerConfig.getString("licenseUrl"))
       ))
 
-    //val storageDir = Paths.get(conf.getString("workspace.storageDir"))
-    //Files.createDirectories(storageDir)
-    //val workspaceDAO = new FileSystemWorkspaceDAO(storageDir)
-    //val entityDAO = NoOpEntityDAO
-    val backingDB = new OrientGraph("memory:rawls") // TODO: use orientdb-dev
-    val workspaceDAO = new GraphWorkspaceDAO(backingDB)
-    val entityDAO = new GraphEntityDAO(backingDB)
-    val service = system.actorOf(RawlsApiServiceActor.props(swaggerService, WorkspaceService.constructor(workspaceDAO, entityDAO)), "rawls-service")
+    val dataSource = DataSource("memory:rawls", "", "", 0, 30)
```

Comment:
Yeah, it will pull it from the same config file as the swagger stuff. Once this is merged I can add it.